inventory/os.cf

```diff
@@ -21,6 +21,11 @@ _inventory_lsb_found::
 
 # Hard coded values for exceptions / platforms without os-release:
 
+(redhat_5|redhat_6).redhat_pure::
```

Comment:
How does this change convert `RedHatEnterpriseServer 6.7` to `Red Hat Enterprise Linux Server 6.7 (Santiago)`?

Reply:
```
[root@ip-172-31-26-226 ~]# lsb_release -a
LSB Version:    :core-4.0-amd64:core-4.0-ia32:core-4.0-noarch:graphics-4.0-amd64:graphics-4.0-ia32:graphics-4.0-noarch:printing-4.0-amd64:printing-4.0-ia32:printing-4.0-noarch
Distributor ID: RedHatEnterpriseServer
Description:    Red Hat Enterprise Linux Server release 5.11 (Tikanga)
Release:        5.11
Codename:       Tikanga
[root@ip-172-31-26-226 ~]#
```

It uses Description from `lsb_release`, instead of the old `id + Release`.

Reply:
Ah, ha! thanks.